test/functionalTest/harnessFunctions.sh

```diff
@@ -87,6 +87,50 @@ function dMsg()
 }
 
 
+# ------------------------------------------------------------------------------
+#
+# dbInitTenant - 
+#
+function dbInitTenant()
```

Comment:
Could not the old **dbInit** be reused for this?
Accepting either **one** or **two** params and if the second is present, it is simply concatenated to the name of the DB ... ?